data/integration_tests/percolator/IntegrationTest_Percolator_SpeedGraphData.py

```diff
@@ -0,0 +1,174 @@
+# Marcus Andersson - Percolator Project
+# Script that tests the performances of percolator for different data input sizes. Produces a file containing data for graphs.
+# psutil is imported in the case that the script should wait for other integration-test scripts to finish.
+
+pathToOutputData = "@pathToOutputData@"
+pathToTestScripts = "@pathToTestScripts@"
+
+import os
+import sys
+import re
+
+from argparse import ArgumentParser
+import time
+import contextlib
+import subprocess
+from subprocess import Popen, PIPE
+from decimal import Decimal
+import pathlib
+import datetime
+from pathlib import Path
+import urllib
+from urllib.request import urlopen, Request
+from urllib.error import URLError, HTTPError
+
+import matplotlib.pyplot as plt
+import pandas as pd
+from itertools import (takewhile,repeat)
+
+testScriptPath = pathToTestScripts + "/IntegrationTest_Percolator_Speed.py"
+minNumLines = -1 #The amount of input lines that also determine the step size between tests. Tests are made with increasing input size determined by number of lines with PSMs.
```

Comment:
If an in-line comment is longer than a few words, it's better to pull it out onto its own line(s).